prebuilt/common/bin/chrootinit

```diff
@@ -2,41 +2,43 @@
 # Initial chroot setup
 
 if [ ! -x /system/bin/getprop ]; then
-  printf "Unable to execute /system/bin/getprop (needed for device detection)\n"
+  /system/bin/toybox printf "Unable to execute /system/bin/getprop (needed for device detection)\n"
   exit 1
 fi
 
-if grep -q squashfs /proc/filesystems; then
+if /system/bin/toybox grep -q squashfs /proc/filesystems; then
   if [ ! -f /sdcard/Android/data/com.pwnieexpress.android.pxinstaller/files/stockchroot.sfs ]; then
-    printf "Unable to read stockchroot.sfs\n"
+    /system/bin/toybox printf "Unable to read stockchroot.sfs\n"
     exit 1
   else
     chroot_file="/sdcard/Android/data/com.pwnieexpress.android.pxinstaller/files/stockchroot.sfs"
   fi
 else
-  printf "No squashfs support present\n"
+  /system/bin/toybox printf "No squashfs support present\n"
   exit 1
 fi
 
 # Error fast
 set -e
 
 # Unpack chroot
-busybox mkdir -p /data/local/kali
-busybox mkdir -p /data/local/kali-tmp
-busybox mount -t auto "${chroot_file}" /data/local/kali-tmp
-busybox cp -a /data/local/kali-tmp/* /data/local/kali
-busybox umount /data/local/kali-tmp
-busybox rm -rf /data/local/kali-tmp
+/system/bin/toybox mkdir -p /data/local/kali
+/system/bin/toybox mkdir -p /data/local/kali-tmp
+/system/bin/toybox cp "${chroot_file}" /data/local
+/system/bin/toybox mount -t auto /data/local/stockchroot.sfs /data/local/kali-tmp
+/system/bin/toybox cp -a /data/local/kali-tmp/* /data/local/kali
+/system/bin/toybox umount /data/local/kali-tmp
+/system/bin/toybox rm -rf /data/local/kali-tmp
+/system/bin/toybox rm /data/local/stockchroot.sfs
```

Comment:
It's not being wiped. It's being copied from the /sdcard for mounting purposes, then cleared from /data/local. It's there for the reset.